### ISSUE TITLE
ESP32?

### ISSUE THREAD

**Issue by bill-orange** [datacute/DoubleResetDetector#9]
Will this work in an ESP32?

**Comment by datacute** [comment]
No, sorry, the hardware features that I make use of, are different on the ESP32.
The concept is probably able to be re-implemented for the ESP32 using the same DoubleResetDetector API, but I don't have any current plans to do so.
